docs/ldml/tr35.md

```diff
@@ -1707,15 +1707,71 @@ For case 2, the children and parent share the same primary language, but the reg
 <parentLocale parent="es_419" locales="es_AR es_BO … es_UY es_VE"/>
 ```
 
-There are certain components that require addenda to the common parent fallback rules. For example, in collation, for a locale like `zh_Hant` in the example above, the `parentLocale` element would dictate the parent as `root` when referring to main locale data, but for collation data, the parent locale would still be `zh`, even though the `parentLocale` element is present for that locale. To address this, components can have their own fallback rules that inherit from the common rules and add additional parents that supplement or override the common rules:
+There are certain components that require addenda to the common parent fallback rules. For a locale like `zh_Hant` in the example above, the `parentLocale` element would dictate the parent as `root` when referring to main locale data, but for collation data, the parent locale should still be `zh`, even though the `parentLocale` element is present for that locale. To address this, components can have their own fallback rules that inherit from the common rules and add additional parents that supplement or override the common rules:
 
 ```xml
 <parentLocales component="segmentations">
   <parentLocale parent="zh" locales="zh_Hant"/>
 </parentLocales>
 ```
 
-The above data would mean that in the "segmentations" component, `zh_Hant` should inherit from `zh` instead of from `root`.
+Logically, component-specific parent locales should be merged with the common parent locales as if merging maps with children as the keys and parents as the values. For example, consider the following XML:
```

Comment:
I think this can work, but needs to be explicit about the order of the merge. If I'm merging normal into collations as a map, then usually the later one wins. But that's not what you want in this case.

Reply:
> as if merging maps with children as the keys and parents as the values. 

Maybe: 

as if merging maps with children as the keys and parents as the values, and retaining the component-specific mapping whenever there are duplicate keys.